examples/migrate_forseti/tutorial.md

```diff
@@ -0,0 +1,362 @@
+# Migrate Forseti from the Python Installer
+
+## Introduction
+
+<walkthrough-tutorial-duration duration="30"></walkthrough-tutorial-duration>
+
+
+This guide explains how to migrate a Forseti deployment from the
+deprecated Python Installer to the new Terraform module.
+
+Completing this guide will also result in a Forseti deployment upgraded to the most recent version.
+
+If you have any
+questions about this process, please contact us by
+e-mail at discuss@forsetisecurity.org or on
+[Slack](https://forsetisecurity.slack.com/join/shared_invite/enQtNDIyMzg4Nzg1NjcxLTM1NTUzZmM2ODVmNzE5MWEwYzAwNjUxMjVkZjhmYWZiOGZjMjY3ZjllNDlkYjk1OGU4MTVhZGM4NzgyZjZhNTE).
+
+## Prerequisites
+
+Before you begin the migration process, you will need:
+
+- A Forseti deployment of at least v2.18.0.  You must follow the
+  [upgrade guide](https://forsetisecurity.org/docs/latest/setup/upgrade.html) to be at least v2.18.0.
+- A version of the
+  [Terraform command-line interface](https://www.terraform.io/downloads.html)
+  in the 0.12 series.
+- The domain and ID of the Google Cloud Platform (GCP) organization in
+  which Forseti is deployed.
+- The ID of the GCP project in which Forseti is deployed.
+- The suffix appended to the names of the Forseti resources; this is
+  likely a string of seven characters like a1b2c3d.
+- A service account for the organization with the
+  [roles required by the Terraform module](https://registry.terraform.io/modules/terraform-google-modules/forseti/google/5.0.0#iam-roles).
+- A
+  [JSON key file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys)
+  for the service account.
+- If you are an Org Admin in the organization in which you deploying Forseti, a separate Service Account and Key are recommended,
+  but not required.
+- **Strongly recommended out of an overabundance of caution:** A backup of your current state.
+  - In the Forseti Server's GCS Bucket
+    - Scanner rules
+    - Server config file
+    - Scanner Violations
+    - Inventory summary
+  - [CloudSQL database](https://cloud.google.com/sql/docs/mysql/backup-recovery/backups)
+
+If you deployed Forseti in a shared VPC then you will also need:
+
+- The ID of the GCP project in which the shared VPC is hosted.
+- The ID of the shared VPC network in which Forseti is deployed.
+- The ID of the subnetwork from the shared VPC network in which Forseti is deployed.
+
+
+## Configure gcloud
+In the Cloud Shell, configure `gcloud` to use the GCP in which Forseti is deployed.
+```bash
+gcloud config set project PROJECT_ID
```

Comment:
You can use {{project_id}} to automatically populate the user's currently selected project. This can be updated in a few other places in the tutorials, not a big deal, but it is nice.